lib/push.js

```diff
@@ -44,34 +44,33 @@ module.exports = function (log, db) {
             if (device.pushCallback) {
               // send the push notification
               log.increment(LOG_UPDATE_INCREMENT_SEND)
-              request.post({
-                url: device.pushCallback,
-                headers: {
-                  'ttl': '0'
-                }
-              }, function (err, response) {
-                if (err) {
+              webpush.sendNotification(device.pushCallback, {
+                'TTL': 0
```

Comment:
They use it as a number in their tests, but I don't see how I wouldn't work with a string